Assignment/Program.cs

```diff
@@ -1,7 +1,20 @@
-﻿namespace Assignment {
-   internal class Program {
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello world!");
+﻿for (; ; ) {
+   Console.Write ("Enter word or sentence : ");
+   string word = Console.ReadLine ().ToLower ().Replace (" ", "");
+   int wordCount = word.Length, count = 0;
+   if (wordCount > 0) {
+      for (int i = 0; i < word.Length; i++) {

```

Comment:
Just a string reverse should do. Please simplify this program.

Reply:
ok